contrib/functions/go/delete-annotations/README.md

```diff
@@ -0,0 +1,34 @@
+# delete-annotations
+
+## Overview
+
+<!--mdtogo:Short-->
+
+Deletes the supplied annotation keys from the resource(s) in a package. It can delete multiple annotations provided as a comma separated string as part of the function config (usage instructions below).
```

Comment:
The 2nd sentence is not necessary here. It belongs in the Usage section.

Reply:
Acked